app/views/deals/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 id= "deals_header"> All deals </h1>
+<div class="jumbotron">
+  <div class="container text-center">
+    <p>Total_transaction: $<%= @total_deal %></p>
+
+        <% @deals.each do |deal| %>
+            <p>Deal name: <%= deal.name %> </p>
+            <p>Deal amount: $<%= deal.amount %> </p> <br>
+        <%end %>
```

Comment:
- [ ] For a given category, the list of transactions should be ordered by the most recent as per the project requirements.

Reply:
@medaminedev66 You can check my deals controller to see that I have ordered the deals by the most recent